addons/survey/static/src/js/survey_form.js

```diff
@@ -13,8 +13,8 @@ var _t = core._t;
 publicWidget.registry.SurveyFormWidget = publicWidget.Widget.extend({
     selector: '.o_survey_form',
     events: {
-        'change .o_survey_form_choice_item': '_onChangeChoiceItem',
-        'click .o_survey_matrix_btn': '_onMatrixBtnClick',
+        'click .o_survey_form_choice_item': '_onChoiceItemClick',
+        'click .o_survey_matrix_btn': '_onChoiceItemClick',
```

Comment:
Wait, is "_onKeyDown" still working, considering we are triggering change event ?